app/Notifications/Mship/DiscordInvite.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Notifications\Mship;
+
+use App\Models\Mship\Account;
+use App\Notifications\Notification;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Support\Facades\Cache;
+
+class DiscordInvite extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail', 'database'];
```

Comment:
if you're writing to the db, I think you need an array representation of the notification

Reply:
There is, its just an empty array (similar to some of the other notifications we have, it just doesn't save any attributes)